README.md

```diff
@@ -3,54 +3,134 @@
+1. Set up `redhataccess/pantheon` as the upstream:  
+       $ cd pantheon
+       $ git remote add upstream git@github.com:redhataccess/pantheon.git
+
+### Creating a working branch
+
+Whenever you work on a new issue, you must create a new working branch based on the latest version of the upstream master branch.
+
+1. Ensure you are on master  
+       $ git checkout master
+1. Ensure your fork is up to date  
+       $ git pull upstream master
+1. Create a working branch based on the issue in JIRA:  
+       $ git checkout -b FCCEUD<ID#>
+
+### Creating a pull request and completing review
+
+When your work is ready to be reviewed and merged, create a pull request.
+
+1. Push your working branch to your fork:  
+       $ git push -u origin <branch_name>
+1. From the repository page in GitHub, click **New pull request**.
+1. Select your working branch from the compare list.
+1. Add `WIP` to the title of the pull request.
+1. Add the **awaiting tech review** label to the pull request.
+1. In the pull request comment field, enter `@redhataccess/pantheon-developers Please review for technical completeness and accuracy`.
+1. Click **Create new pull request**.
+
+For code pull requests, one or more developers review the pull request. For documentation pull requests, the developers review the pull request for technical accuracy and documentation team members review the pull request for clarity and consistency and compliance with necessary standards.
+
+### The review process
+
+Both the technical review and peer review processes take place in pull requests in GitHub.
+
+After creating and labeling a pull request as outlined above, the developers review the pull request and add comments regarding technical accuracy. Writers receive a notification that comments have been added via email, and when all comments have been addressed, the developers change the label from **awaiting tech review** to **tech review passed**.
+
+When technical review is complete, writers click the **Reviewers** gear icon and select the name of a team member to request peer review. Writers receive a notification that comments have been added via email, and when all comments have been addressed, the reviewer clicks **Review changes > Approve** from the **Files changed** tab of the pull request to approve the changes and the pull request.
+
+### Merging a pull request
+
+When you have addressed all technical review and peer review comments, notify the developers to accept the pull request.
+
+1. Remove `WIP` from the title of the pull request.
+1. Click **Request Review** and enter `@redhataccess/pantheon-developers`.
+
+The developers check that the **Tech review passed** label has been added to the pull request and peer pull request approval provided, then accept it.
+
+## Installing Pantheon
+
+### Prerequisites
+* Podman
+* Buildah
+* Java
+
+### Building the application
+_(All commands from here on will be assumed to be run from the project's root directory)_
```

Comment:
Does this note belong here, or does it belong under the "Installing Pantheon" heading?

Reply:
Building the application is a sub-heading of Installing Pantheon. (This is actually out of scope of what I wrote.)